testutil/dbustest.go

```diff
@@ -28,27 +28,10 @@ import (
 	"path/filepath"
 
 	"github.com/godbus/dbus"
-
 	. "gopkg.in/check.v1"
-)
 
-func dbusSessionBus() (*dbus.Conn, error) {
-	// the test suite *must* use a private connection to the bus to avoid
-	// breaking things for code that might use a shared connection
-	conn, err := dbus.SessionBusPrivate()
-	if err != nil {
-		return nil, err
-	}
-	if err := conn.Auth(nil); err != nil {
-		conn.Close()
-		return nil, err
-	}
-	if err := conn.Hello(); err != nil {
-		conn.Close()
-		return nil, err
-	}
-	return conn, nil
-}
+	"github.com/snapcore/snapd/dbusutil"
```

Comment:
I haven't seen any import errors so far.  Long term, I think it would be best if godbus itself provided functions to create usable private connections directly, which would remove the need for this helper.